### PR TITLE
Sorting the examples and excitation models alphabetical

### PR DESCRIPTION
**Pull request by dietmarw**
The examples were difficult to navigate. This PR puts them and the excitation models in a alphabetical order.

### CODE REVIEW

File: OpenIPSL/Electrical/Controls/PSSE/ES/package.order [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ConstantExcitation
-ESST4B
-EXNI
-ST5B
-IEEEX1
-SEXS
 ESDC1A
 ESDC2A
+ESAC1A
```
Comment (marcelofcastro) [inline OpenIPSL/Electrical/Controls/PSSE/ES/package.order:4]: if they are alphabetically, I believe ESAC1A should be the first one, not ESDC1A

Comment (marcelofcastro) [reply OpenIPSL/Electrical/Controls/PSSE/ES/package.order:4]: Fixed in posterior commit.

File: OpenIPSL/Examples/Controls/PSSE/ES/package.order [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-SCRX
 EXAC1
+EXAC2
 EXST1
-ESST1A
-IEEEX1
-ESAC1A
-ESDC1A
-ESDC2A
+EXNI
```
Comment (marcelofcastro) [inline OpenIPSL/Examples/Controls/PSSE/ES/package.order:10]: EXNI should come before EXST1

**Comment by dietmarw** [comment]
Should be fine now.

File: OpenIPSL/Electrical/Controls/PSSE/ES/package.order [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+ESAC2A
```
Comment (marcelofcastro) [inline OpenIPSL/Electrical/Controls/PSSE/ES/package.order]:
```suggestion
ESAC1A
ESAC2A
ESDC1A
ESDC2A
```

File: OpenIPSL/Examples/Controls/PSSE/ES/package.order (hunk 2 above)
Comment (dietmarw) [reply OpenIPSL/Examples/Controls/PSSE/ES/package.order:9]: fixed in cdc04d4

Comment (marcelofcastro) [inline OpenIPSL/Examples/Controls/PSSE/ES/package.order:9]:
```suggestion
EXST1
```

File: OpenIPSL/Electrical/Controls/PSSE/ES/package.order (hunk 3 above)
Comment (marcelofcastro) [reply OpenIPSL/Electrical/Controls/PSSE/ES/package.order]: so I believe I was too late on this suggestion right?

Comment (dietmarw) [reply OpenIPSL/Electrical/Controls/PSSE/ES/package.order]: Yes. Basically that. I had already fixed in locally and pushed it.

**Review by marcelofcastro — APPROVED** [review]
Files are correct and merge is approved

**Pull request merged**